src/api/language.js

```diff
@@ -1,3 +1,9 @@
+import { allLanguages } from 'utils'
+import en from '../../i18n/en.json'
```

Comment:
Relatives paths like these okay in the codebase? Only saw one being used here:

https://github.com/wikimedia/wikipedia-kaios/blob/master/src/index.js#L7

Reply:
It works but it's pretty ugly. You can setup an [alias](https://github.com/wikimedia/wikipedia-kaios/blob/master/webpack.config.js#L51) to make it nicer.